packages/webpack/src/runtime/runtimeBuilder.js

```diff
@@ -0,0 +1,157 @@
+// @ts-ignore
+const { RUNTIME_KEY } = require('../ENUM.json')
```

Comment:
can fix the ts error by adding the file to `webpack/tsconfig.json`
`"include": ["src", "src/ENUM.json"],`
(or could include all json files in the project)